docs/build/program/apis/robot.md

```diff
@@ -170,6 +170,313 @@ For the full list of methods, see the [Viam Python SDK documentation](https://py
+  Must be a valid UUID.
+
+**Returns:**
+
+- [(RobotClient.Options)](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.Options): Options for connecting the `RobotClient`.
+
+```python
+api_key = "your_api_key"
+api_key_id = "valid_uuid"
+opts = RobotClient.Options.with_api_key(api_key, api_key_id)
+
+return await RobotClient.at_address('ADDRESS FROM THE VIAM APP', opts)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.Options.with_api_key).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### AtAddress
+
+Create a robot client that is connected to the robot at the provided address.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `address` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Address of the robot. For example, IP address or URL.
+- `options` [(Options)](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.Options): Options for connecting and refreshing.
+
+**Returns:**
+
+- `RobotClient`: The robot client.
+
+```python {class="line-numbers linkable-line-numbers"}
+async def connect():
+    opts = RobotClient.Options.with_api_key(
+        # Replace "<API-KEY>" (including brackets) with your robot's api key
+        api_key='<API-KEY>',
+        # Replace "<API-KEY-ID>" (including brackets) with your robot's api key
+        # id
+        api_key_id='<API-KEY-ID>'
+    )
+    return await RobotClient.at_address('ADDRESS FROM THE VIAM APP', opts)
+
+
+async def main():
+    # Make a RobotClient
+    robot = await connect()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.at_address).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### WithChannel
+
+Create a robot that is connected to a robot over the given channel.
+Any robots created using this method will NOT automatically close the channel upon exit.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `channel` [(ViamChannel)](https://python.viam.dev/autoapi/viam/rpc/dial/index.html#viam.rpc.dial.ViamChannel): The channel that is connected to a robot, obtained by `viam.rpc.dial`.
+- `options` [(Options)](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.Options): Options for refreshing. Any connection options will be ignored.
+
+**Returns:**
+
+- `RobotClient`: The robot client.
+
+```python {class="line-numbers linkable-line-numbers"}
+from grpclib.testing import ChannelFor
+from viam.robot.service import RobotService
+
+async with ChannelFor([RobotService]) as channel:
+      robot = await RobotClient.with_channel(channel, RobotClient.Options())
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.with_channel).
+
+{{% /tab %}}
+{{< /tabs >}}
```

Comment:
documenting these methods here feels weird - these are specific Python library methods that are not part of the API and so would be different across languages. maybe it should be in a different section?